plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -6,6 +6,7 @@
 module Kubernetes
   class DeployExecutor
     WAIT_FOR_LIVE = ENV.fetch('KUBE_WAIT_FOR_LIVE', 10).to_i.minutes
+    WAIT_FOR_PREREQ = ENV.fetch('KUBE_WAIT_FOR_PREREQ', 10).to_i.minutes
```

Comment:
should fallback to WAIT_FOR_LIVE when not set ?
`WAIT_FOR_PREREQ = ENV.fetch('KUBE_WAIT_FOR_PREREQ', WAIT_FOR_LIVE / 1.minute).to_i.minutes`

Reply:
Yeah, makes sense. The code gets a little tricker with the division and conversion, IMHO. Wonder if can be made simpler...

Would this work:
```
WAIT_FOR_LIVE = ENV.fetch('KUBE_WAIT_FOR_LIVE', 10)
WAIT_FOR_PREREQ = ENV['KUBE_WAIT_FOR_PREREQ'] ||  WAIT_FOR_LIVE
WAIT_FOR_LIVE = WAIT_FOR_LIVE.to_i.minutes
WAIT_FOR_PREREQ = WAIT_FOR_PREREQ.to_i.minutes
```

But let me know which one you prefer (or just modify it, if GitHub default lets you change my branch) or if you can think of something simpler :)


Thanks!